### ISSUE TITLE
Conflict: Constraint npm-bcrypt@=0.7.7 is not satisfied by npm-bcrypt 0.7.8_2

### ISSUE THREAD

**Issue by ofermalbin** [okland/accounts-phone#3]
meteor add okland:accounts-phone
 => Errors while adding packages:             
                                              
While selecting package versions:
error: Conflict: Constraint npm-bcrypt@=0.7.7 is not satisfied by npm-bcrypt 0.7.8_2.
Constraints on package "npm-bcrypt":
* npm-bcrypt@=0.7.8_2 <- accounts-password 1.1.1
* npm-bcrypt@=0.7.7 <- okland:accounts-phone 0.0.10

**Comment by yhyuan** [comment]
Hi ofermalbin, 

You may need to remove accounts-password. I found okland:accounts-phone is conflicting with accounts-password.

**Comment by mitesh-mutha** [comment]
Is there any way I can work with this without removing accounts-password ?

**Comment by yhyuan** [comment]
@mitesh-mutha I do not think it will work out. The package is name conflicting with accounts-password. They can not coexist.

**Comment by okland** [comment]
I update the package to use npm-bcrypt 0.7.8_2. see if it fix your problem.
